packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -658,7 +662,7 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
                   onMoreButtonClicked={onMoreButtonClicked}
                   isCaptionsSupported={(useTeamsCaptions && hasJoinedCall) || hasJoinedCall}
                   /* @conditional-compile-remove(rtt) */
-                  isRealTimeTextSupported={hasJoinedCall && !props.isPstnCall}
+                  isRealTimeTextSupported={hasJoinedCall && !props.isPstnCall && isRealTimeTextSupported}
```

Comment:
nit: is there a way we can do this calculation against the selector to make this simpler?